Packs/Azure-Enrichment-Remediation/Playbooks/Azure_-_Enrichment.yml

```
@@ -2,750 +2,952 @@ id: Azure - Enrichment
 version: -1
 name: Azure - Enrichment
 description: Given an IP address, this playbook enriches Azure Compute, Microsoft Graph User, and IAM information and outputs Azure Compute, Microsoft Graph User, and IAM information.
+inputs:
+- description: Azure IP in Alert
+  key: AzureIP
+  playbookInputQuery:
+  required: false
+  value:
+    complex:
+      accessor: remoteip
+      root: alert
+outputs:
+- contextPath: Azure.Compute
+  description: Azure Compute information.
+  type: unknown
+- contextPath: Azure.Network
+  description: Azure NIC information.
+  type: unknown
+- contextPath: MSGraphIdentity.Role
+  description: Microsoft Graph Role object that contains role ID and user IDs.
+  type: unknown
+- contextPath: Account
+  description: Microsoft Graph User account objects for found user IDs.
+  type: unknown
+- contextPath: MSGraphUser
+  description: Microsoft Graph User objects for found user IDs.
+  type: unknown
+- contextPath: azureAssetHierarchy
```

```suggestion
- contextPath: AzureAssetHierarchy
```
NIT make this camel case as all others are